devlib/target.py

```diff
@@ -288,6 +288,7 @@ def invoke(self, binary, args=None, in_directory=None, on_cpus=None,
                   a ``TimeoutError`` exception will be raised. Set to ``None`` if the
                   invocation should not timeout.
 
+        :returns: output of command.
```

Comment:
Looks good to me, it's the same syntax we use already for example in:
https://github.com/ARM-software/devlib/blob/master/devlib/utils/serial_port.py#L100

:+1:  for merge